build/Dockerfile

```
@@ -1,11 +1,11 @@
 FROM registry.access.redhat.com/ubi7/ubi-minimal
 RUN microdnf update && microdnf clean all
 
-MAINTAINER Percona Development <info@percona.com>
 LABEL name="Percona XtraDB Cluster Operator" \
       vendor="Percona" \
       summary="Percona XtraDB Cluster is an active/active high availability and high scalability open source solution for MySQL clustering" \
-      description="Percona XtraDB Cluster is a high availability solution that helps enterprises avoid downtime and outages and meet expected customer experience."
+      description="Percona XtraDB Cluster is a high availability solution that helps enterprises avoid downtime and outages and meet expected customer experience." \
+      MAINTAINER="Percona Development <info@percona.com>"
```

lowercase according to the example in the official doc - https://docs.docker.com/engine/reference/builder/
```suggestion
      maintainer="Percona Development <info@percona.com>"
```